src/pages/blog/components/post-card-skeleton/PostCardSkeleton.tsx

```diff
@@ -0,0 +1,58 @@
+import { Skeleton } from "@mui/material";
+import { Box, IconButton } from "@mui/material";
+
+import imagePlaceholder from "@/assets/icons/image-placeholder.svg?url";
+import ShareIcon from "@/assets/icons/three-dots-vertical.svg";
+
+import {
+  BoxImageStyled,
+  BoxStyled,
+  ContentBox,
+  ImgStyled,
+  PostContentBox,
+  PostInfoBox,
+  StatsBox,
+} from "@/pages/blog/components/post-card/PostCard.styled";
+import LikeWidgetSkeleton from "@/pages/post/components/like-widget/LikeWidgetSkeleton";
+
+export default function PostCardSkeleton() {
+  return (
+    <BoxStyled>
+      <BoxImageStyled isLoading={true} shouldShowImagePlaceholder={true}>
+        <ImgStyled src={imagePlaceholder} alt="Post image placeholder" />
+      </BoxImageStyled>
+      <ContentBox>
+        <PostInfoBox>
+          <Box display="flex" gap={1}>
+            <Skeleton variant="circular" width={40} height={40} />
+            <Box>
+              <Skeleton width={100} />
+              <Skeleton width={250} />
+            </Box>
+          </Box>
+          <IconButton
+            disableRipple
+            sx={{
+              "&:hover": { backgroundColor: "transparent" },
+              "&:active": { backgroundColor: "transparent" },
+            }}
+            area-label="share"
+          >
+            <ShareIcon />
+          </IconButton>
```

Comment:
Good hob here! But I suggest removing one of skeleton content-like containers to make post with less height

![image](https://github.com/user-attachments/assets/b4fa51cf-5dcb-484f-aa93-ff49439df273)

Reply:
Fixed
Removed skeleton for title
![image](https://github.com/user-attachments/assets/58abd104-b204-49e8-b592-0aa1c835cb40)